ebs/ebs.go

```diff
@@ -334,12 +380,41 @@ func (d *Driver) CreateVolume(req Request) error {
 		if err != nil {
 			return err
 		}
-		volumeSize = *ebsVolume.Size * GB
-		log.Debugf("Found EBS volume %v for volume %v, update tags", volumeID, id)
-		if err := d.ebsService.AddTags(volumeID, newTags); err != nil {
-			log.Debugf("Failed to update tags for volume %v, but continue", volumeID)
+		// Verify that there isn't a more recent snapshot from another AZ
+		otherAZNames, err := d.ebsService.GetOtherAZs()
```

Comment:
Since the calls may be slow - should we have that either in a config or cached ?